packages/salesforcedx-utils-vscode/src/helpers/extractJsonObject.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export function extractJsonObject(str: string) {
```

Comment:
Instead of creating a new file for this functionality, do you think this method could be placed inside of another file with a similar purpose?